src/main/java/sysc4806/project24/mini_survey_monkey/models/CommentResponse.java

```diff
@@ -0,0 +1,19 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.Entity;
+
+@Entity
+public class CommentResponse extends Response {
+
+    private String text;
+
+    public CommentResponse(){}
```

Comment:
All no-arg constructors can be removed from the models, as they'll automatically be generated.

```suggestion
```